content/millennium/dstu2/medications/medication-order.md

```diff
@@ -7,6 +7,36 @@ title: MedicationOrder | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+
+The Medication Order resource provides orders for all medications along with administration instructions for a patient in both the inpatient and outpatient setting (orders/prescriptions filled by a pharmacy and discharge medication orders). This resource does NOT include historical or home medications documented or reported to be taken by patient, significant other or another provider. 
```

Comment:
I see Medication Order and MedicationOrder in the documentation, is one preferable?